src/inputs/input.jsx

```diff
@@ -151,40 +201,36 @@ class Input extends React.PureComponent {
                 }
 
                 this.inputTimer = setTimeout(() => {
-                    if (_.isEmpty(newValue)) {
+                    if (isEmpty(newValue)) {
                         if (required) {
-                            newValue = _.isNumber(min) ? min : (_.isNumber(max)? max : 0);
+                            if (isNumber(min)) {
+                                newValue = min;
+                            } else if (isNumber(max)) {
+                                newValue = max;
+                            } else {
+                                newValue = 0;
+                            }
                         }
                     } else {
-                        newValue = +newValue;
-                        if (_.isNumber(max)) {
+                        newValue += 1;
+
+                        if (isNumber(max)) {
                             newValue = Math.min(max, newValue);
                         }
-                        if (_.isNumber(min)) {
+
+                        if (isNumber(min)) {
                             newValue = Math.max(min, newValue);
                         }
                     }
 
-                    if (_.isUndefined(onChange)) {
-                        this._input.value = newValue;
-                    } else {
-                        onChange(newValue);
-                    }
+                    this.setNewValue(newValue);
                 }, 500);
-
-                if (_.isUndefined(onChange)) {
-                    this._input.value = newValue;
-                } else {
-                    onChange(newValue);
-                }
-            } else {
-                if (_.isUndefined(onChange)) {
-                    this._input.value = newValue;
-                } else {
-                    onChange(newValue);
-                }
             }
 
+            console.log('newValue', newValue);
```

Comment:
Please remove this if the debugging is all done now! :-)

Reply:
Removing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Confirmed; resolving.